js/analyses/getUrlLength.js

```diff
@@ -0,0 +1,20 @@
+/** @module analyses/getUrlLength */
+
+/**
+ * Checks if an URL is too long, based on slug and relative to keyword length.
+ *
+ * @param {string} url The URL to check the length from.
+ * @param {string} keyword The keyword
+ * @param {number} maxSlugLength The maximum length of the slug, as defined in the config
+ * @param {number} maxUrlLength The maximum length of the Url, as defined in the config
+ * @returns {boolean} true if the URL is too long, false if it isn't
+ */
+module.exports = function( url, keyword, maxSlugLength, maxUrlLength ) {
```

Comment:
Shouldn't we be retrieving `maxSlugLength` and `maxUrlLength` from the config instead of as parameters?